content/docs/administration/troubleshoot/databasecomponentunreadyreplicas.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "DatabaseComponentUnreadyReplicas"
+description: ""
```

Comment:
Should we include descriptions here and below (where it is marked as recommended)?